cfme/fixtures/vm.py

```diff
@@ -10,12 +14,49 @@
 
 
 def _create_vm(request, template, provider, vm_name):
+    doc_str = parser.get_docstrings_in_file(request.node.fspath.strpath)
+    merged_str = parser.merge_docstrings(doc_str)
+    regex = re.compile(r"\[.*\]")
+    if isinstance(request.node, pytest.Function):
+        assignee = (merged_str[regex.sub('', get_nodeid_full_path(request.node))]
+                    .get('assignee', ''))
+    else:
+        # Fetch list of tests in the module object
+        test_list = [
+            item
+            for item in dir(request.module)
+            if ('test_' in item[:5]) and not ('test_requirements' == item)
+        ]
+        # Find out assignee for each test in test_list
+        assignee_list = list()
+        for test in test_list:
+            try:
+                assignee_list.append(merged_str['::'.join([request.node.fspath.strpath, test])]
+                    .get('assignee', ''))
+            except KeyError:
+                continue
```

Comment:
does this need to be in a `Try/Except` block? If you are doing a .`get()` it should not throw a `KeyError`?

Reply:
`merged_str['::'.join([request.node.fspath.strpath, test]` this could throw `KeyError` right?